### ISSUE TITLE
contentFile option does not work

### ISSUE THREAD

**Issue by AndrewBresee** [contentful/contentful-import#25]
I am trying to upload content from a file that was just written to by contentful-export

**Observed**
exportDir properly writes to the file in contentful-export, but there is no way for "contentFile" to be used. 

In contentful-import/lib/index.js:
```
export default function runContentfulImport (usageParams) {
  let {opts} = usageParams
  if (!opts) {
    opts = {}
    opts.destinationSpace = usageParams.spaceId
    opts.destinationManagementToken = usageParams.managementToken
    opts.content = usageParams.content
  }
  const clients = createClients(opts)
  opts.content.webhooks = opts.content.webhooks || []
```

contentFile is never handled. 

**Steps to recreate**
Set import params as follows:
```
      let cloneToOptions = {
          version: true,
          "spaceId": config.cloneToExistingSpace,
          "managementToken": config.uploadToContentfulManagementToken,
          "contentFile": uploadFile,
          // "content": foundContent
        }
```
"content" is not being used because it is being read from the file . "uploadFile" in this case is the relative path to the file created from contentful-export. 

Here, the error 
```
 opts.content.webhooks = opts.content.webhooks || [];
                                      ^
TypeError: Cannot read property 'webhooks' of undefined
```
is thrown because uploadFile is not being considered.

**Comment by AndrewBresee** [comment]
It seems like bin/contentful-import the "import tool" is never called. So usageParams `var usageParams = require('../dist/usageParams')` are never properly set.

**Comment by Khaledgarbaya** [comment]
This is a bug when using contentful-import as a node module, I will include the fix in the next beta release. Thanks @AndrewBresee

**Comment by axe312ger** [comment]
fixed in #24

**Comment by AndrewBresee** [comment]
Just as a note, I was able to get around this by importing the file, parsing it, and using it as content:

```

  fs.readFile(uploadFile, (err, contentJSON) => {
          let cloneToOptions = {
            version: true,
            spaceId: config.cloneToExistingSpace,
            managementToken: config.uploadToContentfulManagementToken,
            // contentFile: uploadFile,
            content: JSON.parse(contentJSON)
          }

```
